server/src/ledger.rs

```diff
@@ -58,10 +59,10 @@ pub fn get_stake_weight(
         None => anyhow::bail!("Error: Account not found in ledger."),
     };
 
-    let balance = account.balance.parse::<f64>().unwrap_or(0.00);
+    let balance = account.balance.parse::<Decimal>().unwrap_or(Decimal::new(0, 2));
```

Comment:
Number of decimal points should be consistent to what Mina supports: 9 decimal places.